documentation/index.html

```diff
@@ -256,6 +256,62 @@ <h3 class="content-heading">Getting Started</h3>
                 </div>
                 <hr>
             </div>
+
+            <!-- alert component -->
+            <div class="alert-container">
+                <p class="content-sub_heading">Alert</p>
+                <p class="sub-heading-description">Alerts are mostly used to communicate a state that affects a system, feature or a page!</p>
+                <p class="sub-heading-description">You can view the four different types of alert messages given below:</p>
+                <div class="display-alert">
+                    <div class="alert-item alert-success">
+                        <i class="alert-icon fa fa-check-circle"></i>
+                        <p>Application saved successfully!</p>
+                    </div>
+                </div>
+                <div class="display-alert">
+                    <div class="alert-item alert-info">
+                        <i class="alert-icon fa fa-info-circle"></i>
+                        <p>This is an info alert</p>
+                    </div>
+                </div>
+                <div class="display-alert">
+                    <div class="alert-item alert-warning">
+                        <i class="alert-icon fa fa-exclamation-triangle"></i>
+                        <p>This is a warning alert</p>
+                    </div>
+                </div>
+                <div class="display-alert">
+                    <div class="alert-item alert-error">
+                        <i class="alert-icon fa fa-exclamation-circle"></i>
+                        <p>This is an error alert</p>
+                    </div>
+                </div>
+                <div class="code-snippet">
+                    Use class name as "alert-item" and pick the another class name for the type of alert you want to use as mentioned below:
+                    <pre id="myInput">
+                        <code class="language-html">&lt;div class=&quot;alert-item alert-success&quot;&gt;
+        &lt;i class=&quot;alert-icon fa fa-check-circle&quot;&gt;&lt;/i&gt;
+        &lt;p&gt;Application saved successfully!&lt;/p&gt;
+&lt;/div&gt;
+                        
+&lt;div class=&quot;alert-item alert-info&quot;&gt;
+        &lt;i class=&quot;alert-icon fa fa-check-circle&quot;&gt;&lt;/i&gt;
+        &lt;p&gt;This is an info message&lt;/p&gt;
+&lt;/div&gt;
+                        
+&lt;div class=&quot;alert-item alert-warning&quot;&gt;
+        &lt;i class=&quot;alert-icon fa fa-check-circle&quot;&gt;&lt;/i&gt;
+        &lt;p&gt;This is a warning message&lt;/p&gt;
+&lt;/div&gt;
+
+&lt;div class=&quot;alert-item alert-error&quot;&gt;
+        &lt;i class=&quot;alert-icon fa fa-check-circle&quot;&gt;&lt;/i&gt;
+        &lt;p&gt;This is an error message&lt;/p&gt;
+&lt;/div&gt;
+                        </code>
+                    </pre>
```

Comment:
Use formatting

Reply:
I have to do it this way only because it's used to display the code snippets.